tasks.py

```diff
@@ -216,8 +220,22 @@ def notify(ctx):
         username=env.get_credential('FEC_SLACK_BOT', 'fec-bot'),
     )
 
-def run_migrations(ctx, space):
-    print("\nMigrating database...")
-    jdbc_url = to_jdbc_url(os.getenv('FEC_MIGRATOR_SQLA_CONN_{0}'.format(space.upper())))
+@task
+def create_sample_db(ctx):
+    """
+    Load schema and data into the empty database pointed to by $SQLA_SAMPLE_DB_CONN
+    """
+
+    print("Loading schema...")
+    db_conn = os.getenv('SQLA_SAMPLE_DB_CONN')
+    jdbc_url = to_jdbc_url(db_conn)
+    run_migrations(ctx, jdbc_url)
+    print("Schema loaded")
+    print("Loading sample data...")
+    subprocess.check_call(
+        ['psql', '-v', 'ON_ERROR_STOP=1', '-f', 'data/sample_db.sql', db_conn],
```

Comment:
Oh nice, will this just halt as soon as the first error is encountered?  Although it seems to halt anyway as soon as it hits something.

We also ran into a semi-related issue where the migrations took a long time in CircleCI, causing the cloud.gov token (used for the `cf` CLI tool) to timeout and error out of the rest of the deployment.  I have an idea on how to fix that:  wrap the call to check on the status of the app after migrating (the `cf app api` call) with a try/except and if the call fails, login again and then resume.

We can fix that in a separate PR though, but a broader question came up regarding how long a build can run for in CircleCI (there is a 10 minute timeout, at least by default, when there's no output).  Just wanted to flag it for your attention while it's in the back of my mind.

Reply:
Yes, that is correct. Otherwise, if there are multiple errors during loading, you'll see it going through all of the same and it may be inconvenient to view earlier ones.

Reply:
Excellent, thanks!